bfvmm/include/intrinsics/portio_x64.h

```diff
@@ -39,13 +41,17 @@ namespace portio
     using port_addr_type = uint16_t;
     using port_8bit_type = uint8_t;
     using port_16bit_type = uint16_t;
+    using port_32bit_type = uint32_t;
 
     template<class P, class = typename std::enable_if<std::is_integral<P>::value>::type>
     auto inb(P port) noexcept { return __inb(gsl::narrow_cast<port_addr_type>(port)); }
 
     template<class P, class = typename std::enable_if<std::is_integral<P>::value>::type>
     auto inw(P port) noexcept { return __inw(gsl::narrow_cast<port_addr_type>(port)); }
 
+    template<class P, class = typename std::enable_if<std::is_integral<P>::value>::type>
+    auto inq(P port) noexcept { return __inq(gsl::narrow_cast<port_addr_type>(port)); }
```

Comment:
Should this be ind or inl instead of inq? I thought q usually stood for 8 bytes.